File: NumberOfEvenOdd.java
public class NumberOfEvenOdd {
	public static void main (String[] args) {

	int evenNumber = 0 ;
	int oddNumber = 0 ;
	int number = 1 ;

	while ( number <= 101 ) {
		if (number % 2 == 0) {
			evenNumber += 1; }
		else
			oddNumber += 1;
	}
	number += 1;
	
	System.out.printf("The number of even numbers is: %d%n", evenNumber) ;
	System.out.printf("The number of odd numbers is %d", oddNumber) ;

}

}

File: Number_of_even_odd.py
even_number = 0
odd_number = 0

for num in range(1, 101) :
	if num % 2 == 0:
		even_number += 1
	else :
		odd_number += 1

print ('The number of even numbers is: ', even_number)
print('The number of odd numbers is: ', odd_number)